gui.py

```
@@ -0,0 +1,550 @@
+
+    def _run_status(self):   self._run(["status"])
+
+    def _run_list(self):
+        args = ["list"]
+        hw = self._hw(self.hw_list_cb)
+        if hw: args += ["--hardware", hw]
+        self._run(args)
+
+    def _run_download(self):
+        ver = self._val(self.dl_ver)
+        if not ver: self._warn("Version is required."); return
```

Ruff (0.15.9)

[error] Multiple statements on one line (colon)

(E701)